Cepave/transfer
# [OWL-456] Implement nqmRpc mechanism, which support Cassandra

[OWL-456] Implement nqmRpc mechanism, which support communication with Cassandra.
## How to test:

1 Run the transfer
2 Simulate pkt from NQM agent by 
   cd test
   debug nqm
3 Check the jsonrpc2 pkt sent out from transfer by
   netcat -l  [port]
   Or  using the jsonRpc2 server  deliberately written for this. See the attachment. 
[main.go.txt](https://github.com/Cepave/transfer/files/223416/main.go.txt)
